### DISCUSSION TITLE
Ability to create/delete l2 bridge domains with topology file.

### DISCUSSION THREAD

**Discussion by riddler63** [srl-labs/containerlab#1581] [Ideas]
Hello,

At the moment if you want to connect some nodes by shared media (bridge) you have to create the bridges manually.
It will be nice if the whole topology, including bridges, can be created and destroyed by deploy/destroy commands.

**Comment by hellt** [comment]
Hi
this is tracked here https://github.com/srl-labs/containerlab/issues/84
but so far no development committed
